pkg/plugins/runtime/k8s/webhooks/injector/injector.go

```diff
@@ -438,7 +457,46 @@ func (i *KumaInjector) NewInitContainer(pod *kube_core.Pod) (kube_core.Container
 				kube_core.ResourceMemory: *kube_api.NewScaledQuantity(10, kube_api.Mega),
 			},
 		},
-	}, nil
+	}
+
+	if i.cfg.EBPF.Enabled {
+		// container.SecurityContext.Privileged expects to have a reference
+		// to the bool value
+		tru := true
+		bidirectional := kube_core.MountPropagationBidirectional
+
+		container.Resources = kube_core.ResourceRequirements{
+			Limits: kube_core.ResourceList{
+				kube_core.ResourceCPU:    *kube_api.NewScaledQuantity(300, kube_api.Milli),
+				kube_core.ResourceMemory: *kube_api.NewScaledQuantity(200, kube_api.Mega),
+			},
+			Requests: kube_core.ResourceList{
+				kube_core.ResourceCPU:    *kube_api.NewScaledQuantity(100, kube_api.Milli),
+				kube_core.ResourceMemory: *kube_api.NewScaledQuantity(200, kube_api.Mega),
+			},
+		}
```

Comment:
Can you please explain why this is required in a comment?

Reply:
The init container was failing before, because of lacking resources. After deeper investigation I realized it's failing for the first time because of not enough memory, but when killed and rescheduled with more memory everything was going smoothly, to instead of doing this I decided to bump initial resource requests from 10/10 to 20/20.